internal/provider/resource_job_test.go

```diff
@@ -1283,6 +1283,60 @@ func TestAccCloudAutomatorJob(t *testing.T) {
 				resource.TestCheckResourceAttr("cloudautomator_job.test", "ec2_start_backup_job_action_value.0.additional_tags.2.value", "value3"),
 			},
 		},
+		{
+			name:    "EfsStartBackupJobAction",
+			jobName: fmt.Sprintf("tf-testacc-job-%s", utils.RandomString(12)),
+			configFunc: func(resourceName string) string {
+				return fmt.Sprintf(`
+			resource "cloudautomator_job" "test" {
+				name = "%s"
+				group_id = "%s"
+				aws_account_id = "%s"
+
+				rule_type = "webhook"
+
+				action_type = "ec2_start_backup_job"
+				ec2_start_backup_job_action_value {
+				  region                      = "ap-northeast-1"
+					file_system_id              = "fs-abcdefg1234567890"
+				  lifecycle_delete_after_days = 7
+				  backup_vault_name           = "example-vault"
+				  iam_role_arn                = "arn:aws:iam::123456789012:role/example-role"
+
+				  additional_tags {
+					key   = "key1"
+					value = "value1"
+				  }
+
+				  additional_tags {
+					key   = "key2"
+					value = "value2"
+				  }
+
+				  additional_tags {
+					key   = "key3"
+					value = "value3"
+				  }
```

Comment:
タブとスペースが混在しているので、統一するようにお願いします 🙏 

```suggestion
					region                      = "ap-northeast-1"
					file_system_id              = "fs-abcdefg1234567890"
					lifecycle_delete_after_days = 7
					backup_vault_name           = "example-vault"
					iam_role_arn                = "arn:aws:iam::123456789012:role/example-role"

					additional_tags {
						key   = "key1"
						value = "value1"
					}

					additional_tags {
						key   = "key2"
						value = "value2"
					}

					additional_tags {
						key   = "key3"
						value = "value3"
					}
```